assets/src/components/customers/CustomerDetailsMainSection.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {History} from 'history';
+
+import {Box} from 'theme-ui';
+import {Tabs} from '../common';
+import CustomerDetailsCard from './CustomerDetailsCard';
+import CustomerDetailsConversations from './CustomerDetailsConversations';
+
+const {TabPane} = Tabs;
+
+enum TAB_KEYS {
```

Comment:
very minor nit -- i think enums in typescript are typically written in PascalCase (e.g. `TabKeys.Conversations`) https://www.typescriptlang.org/docs/handbook/enums.html